app/models/language.rb

```diff
@@ -1,2 +1,30 @@
 class Language < ActiveRecord::Base
+
+  # create_table :languages do |t|
+  #   t.string :name
+  #   t.string :code
+  #
+  #   t.timestamps
+
+
+  # returns language_id for given code
+  def self.resolve_id(code = nil)
+    code ||= I18n.language_code
```

Comment:
even if 'nil' is explicitly passed in as a param.  i wanted it to default to the default language.